model/manifest_test.go

```diff
@@ -115,6 +165,23 @@ func TestManifestJson(t *testing.T) {
 		Webapp: &ManifestWebapp{
 			BundlePath: "thebundlepath",
 		},
+		UISettings: &PluginUISettings{
+			HeaderText: "theheadertext",
+			FooterText: "thefootertext",
+			Settings: map[string]*PluginUISetting{
+				"thesetting": &PluginUISetting{
+					DisplayName: "thedisplayname",
+					Type:        PLUGIN_CONFIG_TYPE_DROPDOWN,
+					HelpText:    "thehelptext",
+					Options: []*PluginOption{&PluginOption{
+						DisplayName: "theoptiondisplayname",
+						Value:       "thevalue",
+					},
+					},
```

Comment:
I'm really surprised these two equally indented braces pass the style check. Can we either put them on the same line or separate line 176 into two lines?

Reply:
Sure

Reply:
I don't mean to be a stickler over something so minor, but there are two other places that should get the same treatment.

Reply:
Is it really worth me looking for them and updating them? If gofmt doesn't care, then why do we?